### ISSUE TITLE
Firmware menu: the lower-right corner of the Dreamcast logo is missing

### ISSUE THREAD

**Issue by snickerbockers** [washingtondc-emu/washingtondc#24]
![dc_logo_bug](https://user-images.githubusercontent.com/3129682/32366293-bc0e574a-c03a-11e7-917f-90781240eed8.png)

Oddly enough, it appears that this is *not* a depth-buffer precision issue; based on the wireframe display it looks like the second triangle is actually not getting drawn.

![dc_logo_bug_wireframe](https://user-images.githubusercontent.com/3129682/32366339-f19cf1be-c03a-11e7-9267-a1e506956d55.png)

This was apparently introduced by commit 065999eef15a5b826f9d6622e592512a9e9a452b.

**Comment by snickerbockers** [comment]
This patch makes the bug go away.
```
diff --git a/src/hw/pvr2/pvr2_ta.c b/src/hw/pvr2/pvr2_ta.c
index dd63928..7ca6c4a 100644
--- a/src/hw/pvr2/pvr2_ta.c
+++ b/src/hw/pvr2/pvr2_ta.c
@@ -633,11 +633,11 @@ static void on_polyhdr_received(void) {
      * not 100% clear on whether or not it has its own offset header.  That
      * said, I am confident that intensity mode 2 is 32 bits.
      */
-    if (hdr.color_type != TA_COLOR_TYPE_INTENSITY_MODE_2 &&
-        hdr.offset_color_enable && ta_fifo_byte_count != 64) {
-        // need 64 bytes not, 32.
-        return;
-    }
+    /* if (hdr.color_type != TA_COLOR_TYPE_INTENSITY_MODE_2 && */
+    /*     hdr.offset_color_enable && ta_fifo_byte_count != 64) { */
+    /*     // need 64 bytes not, 32. */
+    /*     return; */
+    /* } */
 
     if ((poly_state.current_list == DISPLAY_LIST_NONE) &&
         list_submitted[list]) {
```

**Comment by snickerbockers** [comment]
This has been resolved by b73892700ad2d04f320bbe492b7359fe98424756.
